sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -297,12 +299,9 @@
   {
     T1_Face        face  = (T1_Face)t1face;
 #ifdef __REACTOS__
-    T1_DecoderRec *decoder = malloc(sizeof(T1_DecoderRec));
-    if (!decoder)
-      return FT_THROW( Out_Of_Memory );
-/* Ugly but it allows us to reduce the diff */
-#define decoder (*decoder)
-    {
+    T1_DecoderRec *decoder = malloc(sizeof(*decoder));
```

Comment:
Please restore the NULL check and early return.

Reply:
Restored.